app/src/main/java/com/zestworks/surveys/ui/SurveyListFragment.kt

```diff
@@ -29,12 +30,11 @@ class SurveyListFragment : Fragment() {
             if (!networkRequestDisposible.isDisposed) {
                 networkRequestDisposible.dispose()
             }
+            surveysViewModel.event_takeSurvey(it)
```

Comment:
the naming here is kinda breaking any convention I've ever known about, either in kotlin or java, or did I miss anything?

Reply:
You are right, it's is not a part of convention. I'll change it. 

I've been using it for a while and got into the habit of writing all view -> viewmodel methods with the suffix "event_". It makes it easier to visualise what is being supplied to the viewmodel.

I'm talking about something like this :
![screenshot at mar 14 22-41-04](https://user-images.githubusercontent.com/3718984/37418847-2914caf8-27d9-11e8-9d2f-c2ef484bb8d4.png)

Reply:
https://github.com/prithivraj/Surveys/pull/27